clearnode/metrics/interface.go

```
@@ -10,46 +10,61 @@ import (
+	IncChannelStateSigValidation(sigType core.ChannelSignerType, success bool)
+	IncChannelSessionKeys()
+	IncAppSessionKeys()
 
 	// api/rpc_router
-	IncRPCMessage(msgType rpc.MsgType, method string)                                 // +
-	IncRPCRequest(method, path string, success bool)                                  // +
-	ObserveRPCDuration(method, path string, success bool, durationSecs time.Duration) // +
-	SetRPCConnections(region, origin string, count uint32)                            // +
+	IncRPCMessage(msgType rpc.MsgType, method string)
+	IncRPCRequest(method, path string, success bool)
+	ObserveRPCDuration(method, path string, success bool, durationSecs time.Duration)
```

golangci-lint (2.11.4)

[error] ST1011: var durationSecs is of type time.Duration; don't use unit-specific suffix "Secs"

(staticcheck)